angr/knowledge_plugins/key_definitions/atoms.py

```diff
@@ -67,7 +67,7 @@ def from_argument(
         argument: SimFunctionArgument, arch: Arch, full_reg=False, sp: int | None = None
     ) -> Register | MemoryLocation:
         """
-        Instanciate an `Atom` from a given argument.
+        instantiate an `Atom` from a given argument.
```

Comment:
I think this should be capitalized, possibly add an s at the end?

Reply:
I think it should be capitalized, but this aligns with the other functions in the file:

```
        Create a Register atom.
        Create a MemoryLocation atom,
```

Reply:
updated